lua/packer/util.lua

```diff
@@ -43,6 +43,29 @@ util.join_paths = function(...)
   return table.concat({...}, separator)
 end
 
+util.absolute_path = function(p)
+  return vim.fn.fnamemodify(p, ":p")
+end
+
+util.path_exists = function(p)
```

Comment:
Just curious - what's the rationale for this approach over e.g. `filereadable()`?

Reply:
I want to use some of these functions when we're not in `vim.schedule_wrapped` stuff. I was getting errors when I was trying to do that some of the time, so I figure I can just use the Lua way to do it.

Reply:
Fair enough - I think, though, that using `vim.loop.fs_access` (https://github.com/luvit/luv/blob/master/docs.md#uvfs_accesspath-mode-callback) would be a bit better (should be faster than opening and closing the file, and is more semantically correct).